internals/daemon/daemon.go

```diff
@@ -275,6 +275,8 @@ func (c *Command) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	fmt.Println(r.Context().Value(requestSrcCtxKey))
+
```

Comment:
Just a debug line to make sure the source of the request can be verified. Will be removed.

Reply:
No longer applicable.